e2e/specs/st_button.spec.js

```diff
@@ -108,4 +129,10 @@ describe("st.button", () => {
     cy.get(".stMarkdown").contains("value: False");
     cy.get(".stMarkdown").contains("value from state: False");
   });
+
+  it("should show buttons centered, stretched and with centered text inside", () => {
+    cy.get("div [data-testid='stVerticalBlock']").each((el, idx) => {
+      cy.wrap(el).matchThemedSnapshots(`stretched_buttons_${idx}`);
+    });
+  });
```

Comment:
Is there a way to select only button 5 and the ones below it in this test? I think it'd be sufficient to only add two snapshots (the light and dark mode snapshots of that group of buttons) for this test since all of the recursive snapshots of `stVerticalBlock` are captured by the snapshot of the group

Feel free to ignore this if it seems too annoying to do

Reply:
It's possible, that would be better, but because iterating and developing E2E tests takes so much time, I'd like to leave it as it is, thank you 🙇